src/index.html

```diff
@@ -12,6 +12,149 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form 
+      class="form1" 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="title">
+          <label for="s">Surname: 
```

Comment:
Change "for" attribute to match the input name, its a good practice. Check other similar cases.